### PR TITLE
Corrected a wrong and misleading limitation

### PR DESCRIPTION
**Pull request by cyrusirandoust**
After spending months with Microsoft on behalf of an important public sector organization, Microsoft recognized that the documentation was wrong about the limitation of "100,000 devices and 7 Mb files" which is by essence impossible as the limitation for any OMA-URI XML file in Microsoft back-end is 350,000 characters. 
If you require, I can share with you the official answer from Microsoft Premier support escalated to the product owner. 
The error in the documentation led our client to loose a lot of time and money to find that specific answer. 
Feel free to contact me for any improvement on my pull request.

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -333,9 +333,11 @@ You can generate GUID through online open source, or through PowerShell - [How t
 
 ![image](https://user-images.githubusercontent.com/81826151/159046476-26ea0a21-8087-4f01-b8ae-5aa73b392d8f.png)
 
-### What is the removable storage media limitation for the maximum number of USBs?
+### What is the removable storage media and policy limitations?
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
### What are the removable storage media and policy limitations?
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 2]
```diff
@@ -333,9 +333,11 @@ You can generate GUID through online open source, or through PowerShell - [How t
...
 
-We've validated one USB group with 100,000 media - up to 7 MB in size. The policy works in both Intune and GPO without performance issues.
+Either from the Microsoft Endpoint Manager portal (Intune) or through MS Graph API, the backend call is done through OMA-URI (GET to read or PATCH to update) and therefore the limitation is the same as any OMA-URI Custom Configuration profile in Microsoft which is officialy 350,000 characters for XML files.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
Either from the Microsoft Endpoint Manager portal (Intune) or through Microsoft Graph API, the backend call is done through OMA-URI (GET to read or PATCH to update) and therefore the limitation is the same as any OMA-URI custom configuration profile in Microsoft which is officially 350,000 characters for XML files. 
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 3]
```diff
@@ -333,9 +333,11 @@ You can generate GUID through online open source, or through PowerShell - [How t
 
 ![image](https://user-images.githubusercontent.com/81826151/159046476-26ea0a21-8087-4f01-b8ae-5aa73b392d8f.png)
 
-### What is the removable storage media limitation for the maximum number of USBs?
+### What is the removable storage media and policy limitations?
 
-We've validated one USB group with 100,000 media - up to 7 MB in size. The policy works in both Intune and GPO without performance issues.
+Either from the Microsoft Endpoint Manager portal (Intune) or through MS Graph API, the backend call is done through OMA-URI (GET to read or PATCH to update) and therefore the limitation is the same as any OMA-URI Custom Configuration profile in Microsoft which is officialy 350,000 characters for XML files. 
+    
+As example, if you need two blocks of Entry per user SID to "Allow"/"Audit allowed" specific users and 2 block of Entry at the end to "Deny" all, you will be able to manage 2276 users.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
For example, if you need two blocks of entries per user SID to "Allow"/"Audit allowed" specific users and two blocks of entries at the end to "Deny all", you will be able to manage 2276 users. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cyrusirandoust, kindly commit the suggested changes. Many thanks.

**Comment by cyrusirandoust** [comment]
Hi @JohanFreelancer9 thanks for the review. I just have a doubt about the "Deny all" as only the word "Deny" is an option, but the place it has in my example means "Deny all". It's implicit, but technically can confuse if people read literally.  
Do you think those changes can be pushed today?

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 4]
```diff
@@ -333,9 +333,11 @@ You can generate GUID through online open source, or through PowerShell - [How t
 
 ![image](https://user-images.githubusercontent.com/81826151/159046476-26ea0a21-8087-4f01-b8ae-5aa73b392d8f.png)
 
-### What is the removable storage media limitation for the maximum number of USBs?
+### What are the removable storage media and policy limitations?
 
-We've validated one USB group with 100,000 media - up to 7 MB in size. The policy works in both Intune and GPO without performance issues.
+Either from the Microsoft Endpoint Manager portal (Intune) or through Microsoft Graph API, the backend call is done through OMA-URI (GET to read or PATCH to update) and therefore the limitation is the same as any OMA-URI custom configuration profile in Microsoft which is officially 350,000 characters for XML files. 
+    
+For example, if you need two blocks of entries per user SID to "Allow"/"Audit allowed" specific users and two blocks of entries at the end to "Deny all", you will be able to manage 2276 users.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
For example, if you need two blocks of entries per user SID to "Allow"/"Audit allowed" specific users and two blocks of entries at the end to "Deny" all, you will be able to manage 2276 users. 
```

**Comment by JohanFreelancer9** [comment]
Hi @cyrusirandoust. Thanks for committing the changes and your clarification. I agree with you, so please revert that specific change. Unfortunately, I cannot give you a specific answer as to when your changes will be merged, but today is certainly a possibility. @yogkumgit?

**Review by denisebmsft — APPROVED** [review]
Thank you @cyrusirandoust @JohanFreelancer9

**Pull request merged**